owasp-top10-2017-apps/a5/tictactoe/deployments/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+
+services:
+
+    api:
+        container_name: a5_api
```

Comment:
Just a suggestion, what do you think of using `a5_tictactoe`?

Reply:
Done!